pkg/extension/extension.go

```diff
@@ -69,7 +60,6 @@ func (ext *Extension) Handle(
 	podCopy.Spec.DNSConfig = &corev1.PodDNSConfig{
 		Nameservers: nameservers,
 		Options:     options,
-		Searches:    searches,
```

Comment:
Instead of not updating, I believe this just plain clobbers any search domains instead… which can't be great?

Reply:
good catch, thx